docs/pages/management/operations/ca-rotation.mdx

```diff
@@ -48,7 +48,7 @@ The rotation consists of several phases:
 
 There are two kinds of certificate rotations:
 
-- **Manual:** it is the cluster administrator's responsibility to transition
+- **Manual:** it is the cluster administrators responsibility to transition
```

Comment:
This was correct as-is. In this case, administrator is possessive (the responsibility belongs to them)